src/main/java/com/bravo/user/utility/ValidatorUtil.java

```diff
@@ -60,6 +60,6 @@ private static boolean isDateFilterValid(final DateFilter<?> date){
   }
 
   private static boolean isStringValid(final String string){
-    return string != null && !string.trim().isEmpty();
+    return string != null && !string.trim().isEmpty() && string.matches("[a-zA-Z!*% ]+");
```

Comment:
I like the use of regular expressions here. I don't like that this generic implementation in a shared validator was changed to conform to these validation rules when the rules for another String used in the app could be different. 

I would have preferred to see a unique method that validated the name specifically or an expansion on this generic method to accept an optional RegExp as a parameter.

Reply:
I had a similar thought, but noticed that the only search String search parameters used were id, first name, middle name, and last name, which I thought could have the same validation.